LuaRules/Gadgets/gui_settings.lua

```diff
@@ -0,0 +1,21 @@
+function gadget:GetInfo()
+	return {
+		name	= 'Initial Settings',
+		desc	= 'Handles initial engine settings',
+		author	= 'TurBoss',
+		version	= 'v0.1',
+		date	= 'Oct 25-10-2015 ',
+		license	= 'GNU GPL, v2 or later',
+		layer	= 0,
+		enabled	= true
+	}
+end
+
+if (gadgetHandler:IsSyncedCode()) then
+	return
+end
+
+function gadget:GameSetup(state,ready,playerStates)
```

Comment:
if params aren't used there's no need to list them; i prefer to have function defs minimal, thus :GameSetup() in this case